beta/src/pages/apis/render.md

```diff
@@ -4,16 +4,74 @@ title: render()
+- [Usage](#usage)
+  - [Rendering the root component](#rendering-the-root-component)
+  - [Rendering multiple roots](#rendering-multiple-roots)
+  - [Updating the rendered tree](#updating-the-rendered-tree)
+
+## Reference {/*reference*/}
+
+### `render(reactNode, container)` {/*render*/}
+
+Call `render` to display a React component inside a `container` browser DOM element.
 
 ```js
+const container = document.getElementById('root');
 render(<App />, container);
-render(<App />, container, callback);
 ```
 
-</Intro>
+React will take over managing the DOM inside of the `container`.
+
+An app fully built with React will usually only have one `render` call with its root component.  A page that uses "sprinkles" of React for parts of the page may have as many `render` calls as needed. [See examples below.](#usage)
+
+#### Parameters {/*parameters*/}
+
+* `reactNode`: A *React node* that you want to display. This will usually be a piece of JSX like `<App />`, but you can also pass a React element constructed with [`createElement()`](/TODO), a string, a number, `null`, or `undefined`. 
+
+* `container`: A [DOM element](https://developer.mozilla.org/en-US/docs/Web/API/Element). React will change its content to the rendering output of the first argument.
+
+#### Returns {/*returns*/}
+
+`render` usually returns `null`. However, if the `reactNode` you pass is a *class component*, then it will return an instance of that component.
+
+#### Caveats {/*caveats*/}
+
+* The first time you call `render` with some `container` node, React will clear that node's existing HTML content before rendering the React component into it. If your `container` contains HTML generated by React on the server or during the build, use [`hydrate()`](/TODO) instead, which attaches the event handlers to the existing HTML.
+
+* If you call `render` on the same `container` more than once, React will update the DOM as necessary to reflect the latest JSX you passed. React will decide which parts of the DOM can be reused and which need to be recreated by ["matching it up"](/learn/preserving-and-resetting-state) with the previously rendered tree. Calling `render` on the same `container` again is similar to calling the [`set` function](/apis/usestate#setstate) on the root component: React avoids unnecessary DOM updates.
+
+* If your app is fully built with React, you shouldn't need to use `render` more than in a single place in your app. (You might even not use it at all if a framework does this for you.) If you want to render something in a different part of the DOM tree (for example, a modal or a tooltip), use [`createPortal`](TODO) instead.
+
+---
 
-## Rendering the root component {/*rendering-the-root-component*/}
+### `render(element, container, callback)` {/*render-callback*/}
+
+Same as [`render(element, container)`](#renderreactnode-container).
+
+#### Parameters {/*render-callback-parameters*/}
+
+* `element`: Same as above.
+* `container`: Same as above.
```

Comment:
In the TOC:
<img width="373" alt="Screenshot 2022-02-10 at 9 32 19 AM" src="https://user-images.githubusercontent.com/32865581/153334779-5777713f-eb1d-4234-9c5d-26349bba4948.png">

In the right pane:
<img width="309" alt="Screenshot 2022-02-10 at 9 32 27 AM" src="https://user-images.githubusercontent.com/32865581/153334795-681962a5-32ab-4325-a99e-f6c8a4ad863a.png">